ccd-definition/CaseField.json

```diff
@@ -262,17 +262,18 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "standardDirectionsLabel",
-    "Label": "Upload standard directions.",
-    "FieldType": "Label",
+    "ID": "standardDirections",
+    "Label": "1. Standard directions",
+    "FieldType": "StandardDirections",
```

Comment:
it was specifically asked for these to be grouped to match other tab types by having it in complex type otherwise theres no differentiation, especially with standard directions having no title and UI having it as `upload a file`

Reply:
I guess label would achieve similar result but will not push for it.